demos/serverless/README.md

```diff
@@ -28,6 +28,19 @@ npm install
 npm run deploy -- -r us-east-1 -b <my-bucket> -s <my-stack-name> -a meeting
 ```
 
+#### ChimeSDKMeetings Namespace vs Chime NameSpace
+The Chime and the ChimeSDKMeetings are both Amazon Chime's AWS clients to help builders create the Amazon Chime SDK meeting, attendees, enable transcription features and so on. Going forward, we will be using ChimeSDKMeetings client as the default client.
+
+For local development.....
```

Comment:
Please remove, this line is redundant.

Reply:
Yeah, I added this as an example, but the next comment from Alex should be fine.
https://github.com/aws/amazon-chime-sdk-js/pull/1851/files#r764166752